### ISSUE TITLE
Notification after uploading the file

### ISSUE THREAD

**Issue by DanielZini** [icetee/remote-ftp#636]
When the FTP tree is closed, no return message appears if the file has been uploaded or not.

**Comment by jpxd** [comment]
What behavior do you suggest? Displaying a message for each upload has been suggested already a few times, but I think this would disturb people who use auto-upload on save and save frequently because the upload messages would pile up.
I can think of a configurable option in the package settings, but what kind of notification do you want?

**Comment by breadlesscode** [comment]
A configurable option in the package settings would be awesome =)

**Comment by DanielZini** [comment]
Exactly, a notification that the file has been uploaded or not. Looks like you already have this option now, thanks!

**Comment by DanielZini** [comment]
The notification was very good, but it only appears when uploading files directly from the tree view, when I upload a file open it does not appear.

**Comment by LinzardMac** [comment]
I have the "enable notifications on transfer" option in my settings but I'm wondering where I am supposed to see the notifications. Initially thought I would go to the "notifications" system preferences on my computer but I did not see Atom listed in there as one of the apps that was requesting notification functionality.

**Comment by samifouad** [comment]
@DanielZini @LinzardMac The visual feedback that exists now is watching the bottom left corner in the sync area. When that progress bar and text go away, the transfer is done.

Another way this could work is to have a big banner notification in the top right similar to what appears when you connect. Is that the type of notification you want when you press save on a file?

**Comment by DanielZini** [comment]
Yes, because the ftp tree view is not always open when I upload a file

**Comment by LinzardMac** [comment]
Yea I don't always have the tree view open so something "flashy" that appears and goes away on it's own is what I would like to see. Basically something I can register as "this event took place" from my peripheral vision without having to read or watch closely. Currently my "hot fix" flow uses Atom and "Transmit FTP" which just tosses up a grown notification when the edits are saved in atom and pushed to the server connected via Transmit FTP and that grown notification is fine.

**Comment by icetee** [comment]
Do I know this is implemented?
